src/cogserver/vrt.py

```diff
@@ -0,0 +1,99 @@
+from typing import List, Literal, Optional
+from enum import Enum
+from fastapi import APIRouter, Query, Response
+from titiler.core.factory import FactoryExtension, TilerFactory, BaseTilerFactory, MultiBaseTilerFactory
+from osgeo import gdal
+import tempfile
+
+router = APIRouter()
+
+
+class ResampleAlg(Enum):
+    nearest: str = "nearest"
+    bilinear: str = "bilinear"
```

Comment:
This class used in `create_vrt_from_urls` should also be moved to the same file of `VRTExtension` class.